README.md

```diff
@@ -100,10 +99,22 @@ with docker compose and the frontend locally:
 ```bash
 docker compose up api content-fetch
 cd packages/web
-cp .env.template .env
+cp .env.template .env.local
 yarn dev
 ```
 
+You will need to configure some values in the new `.env.local` file. These are
+the values for running the `web` service directly on your host machine and
+running `api` and `content-fetch` within docker:
+
+```sh
+NEXT_PUBLIC_BASE_URL=http://localhost:3000
```

Comment:
these values work for me, I'm not sure if they're actually _correct_ for more complete testing.